code/controllers/subsystem/lag_switch.dm

```diff
@@ -111,6 +111,19 @@ SUBSYSTEM_DEF(lag_switch)
 				to_chat(world, span_boldannounce("Slowmode for IC/dead chat has been disabled by an admin."))
 		if(DISABLE_NON_OBSJOBS)
 			world.update_status()
+		if(DISABLE_PARALLAX)
+			if (state)
+				to_chat(world, span_boldannounce("Parallax has been disabled for performance concerns."))
+			else
+				to_chat(world, span_boldannounce("Parallax has been re-enabled."))
+
+			for (var/mob/mob as anything in GLOB.mob_list)
+				mob.hud_used?.update_parallax_pref()
```

Comment:
does this remove the relevant background planes from the clients view?

Reply:
https://user-images.githubusercontent.com/35135081/142786193-ea690fe7-0ef6-4633-b1c5-c09d7587e61a.mp4

Does this answer your question, or am I mistaken?

Reply:
yeah